add-on/panels/sidebar.html

```diff
@@ -49,8 +50,7 @@
     <script type="text/javascript" src="shared/js/textChatView.js"></script>
     <script type="text/javascript" src="shared/js/linkifiedTextView.js"></script>
     <script type="text/javascript" src="panels/js/desktopViews.js"></script>
-    <script type="text/javascript" src="panels/js/feedbackViews.js"></script>
```

Comment:
Please add an `XXX Akita` for the feedback views. This is something I've just realised the UX designs are probably missing.